src/search/evaluator.cc

```diff
@@ -97,6 +97,10 @@ void add_evaluator_options_to_feature(plugins::Feature &feature) {
     utils::add_log_options_to_feature(feature);
 }
 
+tuple<std::string, utils::Verbosity> get_evaluator_parameters_from_options(const plugins::Options &opts){
+    return utils::get_log_parameters_from_options(opts);
+}
```

Comment:
Our convention would be to write `string` rather than `std::string` here (for the same reason we write just `tuple`).